### PR TITLE
Update READMEs

### PR DESCRIPTION
**Pull request by leszko**
Update READMEs:
- Add information about how to set up the Minishift environment and run the Docker images
- Remove READMEs in subdirectories (not needed)

Note that: 
1. Information how to run Management Center will be sent in a separate PR
2. Eventually, most of the README documentation and templates should be moved to code samples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,246 @@
...
+Hazelcast::NodeState=ACTIVE
+Hazelcast::ClusterState=ACTIVE
+Hazelcast::ClusterSafe=TRUE
+Hazelcast::MigrationQueueSize=0
+Hazelcast::ClusterSize=1
+```
+
+## Local Docker images
+
+During the development process, a very common use case is to build locally own Docker images and run them on Minishift. E.g. you may want to build on top of the Hazelcast OpenShift image and check if it works, or you may want to create a seprate application and check how it interacts with Hazelcast when deployed together on OpenShift.
```
Comment (Serdaro) [inline README.md]: ...on Minishift. For example, you may want to build..........to create a separate application....

File: README.md [hunk 2]
```diff
@@ -2,10 +2,246 @@
...
+
+In case of Hazelcast Enterprise, make sure that you have a valid license key for Hazelcast Enterprise version. You can get a trial key from [this link](https://hazelcast.com/hazelcast-enterprise-download/trial/).
+
+**1) Setup and start Minishift**
+```
+$ minishift setup-cdk
+$ minishift start
+```
+
+Note that the presented deployment process is done via the `oc` CLI tool, however each of the next steps can be also performed using OpenShift Web Console (accessed by `minishift console`).
```
Comment (Serdaro) [inline README.md]: ....CLI tool, however, each of the....

File: README.md (hunk 1 above)
Comment (leszko) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (leszko) [reply README.md]: Done

**Comment by leszko** [comment]
Thanks for the comments @Serdaro

**Comment by leszko** [comment]
Ok, I'll contact Ben. 

We don't miss any information. Actually it was already duplicated between the sub pages and the main page.

**Comment by bensawyer** [comment]
Do I need to make a website change?

Regards,
– Ben

Benjamin Sawyer
CEO & Consulting Team Lead
Big Yellow Umbrella Marketing Services <http://bigyellowumbrella.com>
Email: ben@bigyellowumbrella.com
Phone: +1 (415) 429-9019 <+141542990197>

This email was sent on behalf of Hazelcast, Inc., a client of Big Yellow
Umbrella, Inc.

On March 14, 2018 at 4:57:35 AM, leszko (notifications@github.com) wrote:

Merged #11 <https://github.com/hazelcast/hazelcast-openshift/pull/11>.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/hazelcast/hazelcast-openshift/pull/11#event-1520753011>,
or mute the thread

**Pull request merged**